src/app/public/src/modules/nav/nav.component.ts

```diff
@@ -13,14 +11,11 @@ import { StacheRouteService } from '../shared';
 export class StacheNavComponent implements OnInit, StacheNav {
   @Input()
   public routes: StacheNavLink[];
-
   @Input()
   public navType: string;
-
   public classname: string = '';
 
-  public constructor(
-    private routerService: StacheRouteService) { }
+  public constructor(private routerService: StacheRouteService) {}
```

Comment:
Know this wasn't you, but noticing it now.  `routerService` is an instance of `StacheRouteService`, we should decide on `route` vs `router`